docs/changelog.rst

```diff
@@ -5,6 +5,8 @@ Unreleased
 ----------
 
 * Update qiskit-ibm-runtime version requirement to 0.22.
+* remove all remote simulators
+* rename ``IBMQLocalEmulatorBackend`` to ``IBMQEmulatorBackend``
```

Comment:
Maybe add that simulatons that used `IBMQEmulatorBackend` previously will now run locally. Just to be explcit.

Reply:
done in [a7a2470](https://github.com/CQCL/pytket-qiskit/pull/301/commits/a7a247060088f8eb09a6785f2af3659df9f73f1c)